internal/pgmonitor/util.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
 )
 
+const (
+	// Where we mount the queries.yaml file
+	QueriesConfigDir = "/opt/crunchy/conf"
```

Comment:
I think it would be good to make this configurable. Currently, deploy dev won't work unless you make a code change. Pulling it from an env var then setting that for deploy-dev in the Makefiles would be helpful